launch_testing/launch_testing/tools/output.py

```diff
@@ -77,3 +82,49 @@ def expected_output_from_file(path):
             return [re.compile(regex) for regex in f.read().splitlines()]
 
     raise RuntimeError('could not find output check file: {}'.format(path))
+
+
+def expect_output(
+    text=None,
```

Comment:
Maybe, it's clearer if you only use keyworded arguments.

Reply:
That was the case originally, but not requiring `text` to be a keyword plays its role when `expect_output` is used as a condition for `ProcessProxy.wait_for_output` (or we force a boilerplate lambda). See [here](https://github.com/ros2/launch/pull/279/files#diff-e6ac5ee5445a9a82cfbe47983f5e4529R66-R84).

Reply:
Makes sense